code/modules/hydroponics/grown/misc.dm

```diff
@@ -137,3 +137,34 @@
 	playsound(src, 'sound/goonstation/misc/fuse.ogg', seed.potency, 0)
 	reagents.chem_temp = 1000 //Sets off the black powder
 	reagents.handle_reactions()
+
+	//cucumber
+	/obj/item/seeds/cucumber
+		name = "pack of cucumber seeds"
+		desc = "contains cucumber seeds"
```

Comment:
That first would should be capitalized and there should be punctuation on the end. Most descriptions are also a bit more interesting than this for seeds, you could consider something like "Pickles in their most primitive state." for example